src/tools/miri/tests/ui.rs

```diff
@@ -270,13 +270,15 @@ regexes! {
     // erase thread caller ids
     r"call [0-9]+"                  => "call ID",
     // erase platform module paths
-    "sys::pal::[a-z]+::"                  => "sys::pal::PLATFORM::",
+    r"\bsys::pal::[a-z]+::"         => "sys::pal::PLATFORM::",
```

Comment:
The `pal` one can be removed now, right? It is subsumed by the 2nd pattern.

Reply:
Oh, right :woman_facepalming:. For some reason, `sys::pal` felt special.